lib/pbench/server/unpack_tarballs.py

```diff
@@ -0,0 +1,110 @@
+import glob
+from logging import Logger
+import os
+from pathlib import Path
+
+from pbench.server import PbenchServerConfig
+from pbench.server.filetree import FileTree
+from pbench.server.utils import get_tarball_md5
+
+
+class Results:
+    def __init__(
+        self,
+        ntotal=0,
+        nsuccess=0,
+    ):
+        self.ntotal = ntotal
+        self.nsuccess = nsuccess
+
+
+class Unpack:
+    LINKSRC = "TO-UNPACK"
+
+    def __init__(self, config: PbenchServerConfig, logger: Logger):
+        """Unpacks Tarball in the INCOMING Directory
+
+        Args:
+            config: PbenchServerConfig configuration object
+            logger: A Pbench python Logger
+        """
+        self.config = config
+        self.logger = logger
+
+    def unpack(self) -> (int, int):
```

Comment:
You're actually returning a `Results` object, apparently, not a tuple. (I also don't think that `(int, int)` as a type hint works until Python 3.10.)

This affects both the type hint and the returns description in the docstring.

Reply:
missed that